packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -264,7 +264,131 @@ describe( 'Navigation', () => {
 		await deleteUser( contributorUsername );
 	} );
 
-	describe( 'placeholder', () => {
+	describe( 'loading states', () => {
+		it( 'does not show a loading indicator if there is no ref to a Navigation post', async () => {
+			await createNewPost();
+			await clickOnMoreMenuItem( 'Code editor' );
+			const codeEditorInput = await page.waitForSelector(
+				'.editor-post-text-editor'
+			);
+			await codeEditorInput.click();
+			const markup = '<!-- wp:navigation /-->';
```

Comment:
Yes it's correct that this is the block markup when inserting a Nav block for the first time. I was trying to test the loading state of the block when it isn't selected. If you insert the block it will be selected which is different.

I could have different tests. One which inserts a block and reloads the page and asserts on the loading results. And another which checks on the behaviour when the block is initially inserted. 

What do you think?

It's all about ensuring we don't see loading indicators in either selected or unselected states.

Reply:
Good point about it not being selected. Maybe add a comment to the test to mention that's the reason behind using the code editor.

Reply:
✅